### PR TITLE
PICARD-991: Enable Acoustid scan action on albums and tracks

### PR DESCRIPTION
**Pull request by zas**
# Summary

* This is a…
    * [ ] Bug fix
    * [x] Feature addition
    * [ ] Refactoring
    * [ ] Minor / simple change (like a typo)
    * [ ] Other
* **Describe this change in 1-2 sentences**:

# Problem

* JIRA ticket (_optional_): [PICARD-991](https://tickets.metabrainz.org/browse/PICARD-991)

# Solution

# Action

### CODE REVIEW

**Comment by samj1912** [comment]
How does this behave with the files on the right pane? Are they retagged with the matched release from acoustid?

**Comment by zas** [comment]
> How does this behave with the files on the right pane? Are they retagged with the matched release from acoustid?

Yes.

File: picard/file.py [hunk 1]
```diff
@@ -531,7 +531,7 @@ def can_edit_tags(self):
 
     def can_analyze(self):
         """Return if this object can be fingerprinted."""
-        return True
+        return not bool(self.metadata["acoustid_id"])
```
Comment (phw) [inline picard/file.py]: `bool()` should not really be needed here.

**Comment by samj1912** [comment]
I agree with @phw.

**Comment by zas** [comment]
It was just to start something about issues related to fingerprinting/scan. I'll close this PR, i let someone propose a more complete PR.

**Pull request closed without merging**